core/lib/vm_interface/src/types/outputs/execution_result.rs

```diff
@@ -118,6 +118,7 @@ pub struct VmExecutionResultAndLogs {
     pub logs: VmExecutionLogs,
     pub statistics: VmExecutionStatistics,
     pub refunds: Refunds,
+    pub new_known_factory_deps: Option<Vec<(H256, Vec<u8>)>>,
```

Comment:
Dumb question: What does this field mean? Could you explain in a doc comment for this field what it's necessary for?